iml-wire-types/Cargo.toml

```diff
@@ -13,6 +13,7 @@ im = {version = "15.0", features = ["serde"]}
 iml-api-utils = {path = "../iml-api-utils", version = "0.3"}
 iml-change = {path = "../iml-change", version = "0.1", optional = true}
 juniper = {git = "https://github.com/graphql-rust/juniper", optional = true}
+lazy_static = "1.4.0"
```

Comment:
It looks like nothing changed in wire-types but dependencies were added.

Reply:
I think it's a leftover, will fix